js/events.js

```diff
@@ -0,0 +1,103 @@
+export function createEventElement(event) {
+  const eventElement = document.createElement("div");
+  eventElement.className = "calendar-event";
+  eventElement.innerHTML = `<strong>${event.title}</strong><br>${event.description}`;
+  const startDateTime = new Date(event.startDateTime);
+  const endDateTime = new Date(event.endDateTime);
+  let startDay = startDateTime.getDay();
+  let dayOfWeek = startDateTime.getDay();
+  const startHour = startDateTime.getHours() + startDateTime.getMinutes() / 60;
+  const endHour = endDateTime.getHours() + endDateTime.getMinutes() / 60;
+  const totalDays = Math.ceil(
+    (endDateTime - startDateTime) / (1000 * 60 * 60 * 24)
+  );
+  console.log("total days " + totalDays);
+  if (totalDays - 1 <= 1 && totalDays != 2) {
+    const calendarCells = document.getElementById("calendarCells");
+    if (startDay === 0) {
+      startDay = 7;
+    }
+    const dayColumn = calendarCells.querySelector(
+      `.cell:nth-child(${startDay})`
+    );
+    if (!dayColumn) {
+      console.error("Day column not found for day:", dayOfWeek);
+      return;
+    }
+    const eventDuration = (endHour - startHour) * 60;
+    eventElement.style.position = "absolute";
+    eventElement.style.top = `${startHour * 60 - 5 * 60}px`;
+    eventElement.style.height = `${eventDuration}px`;
+    dayColumn.appendChild(eventElement);
+  } else {
+    const weekHeader = document.querySelector(".week-header");
+    const eventBar = document.createElement("div");
+    eventBar.className = "multi-day-event";
+    eventBar.id = "multipleDayEvent";
+    eventBar.innerHTML = `<strong>${event.title}</strong>`;
+    const startDayIndex = startDay === 0 ? 7 : startDay;
+    const endDayIndex = startDayIndex + totalDays - 1;
+    eventBar.style.gridColumn = `${startDayIndex + 1} / ${endDayIndex + 2}`;
+    weekHeader.appendChild(eventBar);
+    eventBar.addEventListener("click", () => showEventDetails(event));
+  }
+  eventElement.addEventListener("click", () => showEventDetails(event));
+}
```

Comment:
if this function handles both short and multi day events, let's split it to smaller functions for better readability, like:
renderEvents = (events) => {
   // split events to get short events and multi day events
   // render short events
   // render long events
}

by doing this there should be less ifs, also try to move code to utils when possible